src/symbols.cs

```diff
@@ -0,0 +1,215 @@
+using CoreFoundation;
+using ObjCRuntime;
+using Foundation;
+
+using System;
+
+namespace Symbols {
+	[Watch (10, 0), TV (17, 0), Mac (14, 0), iOS (17, 0), MacCatalyst (17, 0)]
+	[BaseType (typeof (NSObject))]
+	interface NSSymbolEffect : NSCopying, NSSecureCoding { }
+
+	[Watch (10, 0), TV (17, 0), Mac (14, 0), iOS (17, 0), MacCatalyst (17, 0)]
+	[BaseType (typeof (NSObject))]
+	interface NSSymbolContentTransition : NSCopying, NSSecureCoding { }
+
+	[Watch (10, 0), TV (17, 0), Mac (14, 0), iOS (17, 0), MacCatalyst (17, 0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface NSSymbolEffectOptions : NSCopying, NSSecureCoding {
+		[Static]
+		[Export ("options")]
+		NSSymbolEffectOptions CreateOptions ();
+
+		[Static]
+		[Export ("optionsWithRepeating")]
+		NSSymbolEffectOptions CreateOptionsWithRepeating ();
```

Comment:
Same, but also removing `With`:

```suggestion
		NSSymbolEffectOptions CreateRepeating ();
```

and the same for the other Get and Create methods here.

This matches the Swift names better (and also sounds better).